File: src/main/java/cn/xihan/lib/easyhttp/request/OptionsRequest.java
package cn.xihan.lib.easyhttp.request;


import cn.xihan.lib.easyhttp.model.HttpMethod;

/**
 *    author : Android 轮子哥
 *    github : <a href="https://github.com/getActivity/EasyHttp">...</a>
 *    time   : 2021/04/29
 *    desc   : Options 请求
 */
public final class OptionsRequest extends UrlRequest<OptionsRequest> {

    public OptionsRequest() {
        super();
    }

    @Override
    public String getRequestMethod() {
        return HttpMethod.OPTIONS.toString();
    }
}

File: src/main/java/cn/xihan/lib/easyhttp/request/OptionsRequest.kt
package cn.xihan.lib.easyhttp.request

import cn.xihan.lib.easyhttp.model.HttpMethod

/**
 * author : Android 轮子哥
 * github : [https://github.com/getActivity/EasyHttp](https://github.com/getActivity/EasyHttp)
 * time   : 2021/04/29
 * desc   : Options 请求
 */
class OptionsRequest : UrlRequest<OptionsRequest?>() {
    override val requestMethod: String
        get() = HttpMethod.OPTIONS.toString()
}